lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,168 @@ fields:
+    description: The date and/or time that this set of answers were last changed.
+    action: update
+    example: |
+      {
+        "authored": "2019-10-25T14:57:43Z"
+      }
+
+  - name: item
+    required: 'Yes'
+    type: List of BackboneElement
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item
+    description: A group or question item from the original questionnaire for which answers are provided.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93",
+            "text": "Tobacco",
+            "item": [
+              {
+                "linkId": "93-123",
+                "text": "Number of Years:",
+                "answer": [
+                  {
+                    "valueQuantity": "20" 
+                  }
+                ]
+              }
+            ] 
+          }
+        ]
+      }
+
+  - name: item.linkId
+    required: 'Yes'
+    type: string
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93"
+      }
+
+  - name: item.text
+    required: 'No'
+    type: string
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text
+    description: Text that is displayed above the contents of the group or as the text of the question being answered.
+    action: update
+    example: |
+      {
+        "text": "Tobacco"
+      }
+
+  - name: item.item
+    required: 'No'
+    type: List of BackboneElement
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item
+    description: Questions or sub-groups nested beneath a question or group.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93-123",
+            "text": "Number of years:",
+            "answer": [
+              {
+                "valueQuantity": "20"
+              }
+            ]
+          }
+        ]
+      }
+
+  - name: item.item.answer
+    required: 'Yes'
+    type: string | Coding | Quantity
+    url: https://www.hl7.org/fhir/r4/resource-definitions.html#Resource.id
+    description: The respondent's answer(s) to the question.
+    action: update
+    example: |
+      {
+        "answer": [
+          {
+            "valueString": "Text"
+          }
+        ]
+      }
+    example2: |
+      {
+        "answer": [
+          {
+            "valueCoding": {
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/nomenclature",
+              "code": "960439",
+              "display": "Yes"
+            }
+          }
+        ]
+      }
+    example3: |
+      {
+        "answer": [
+          {
+            "valueQuantity": "20"
```

Comment:
valueQuantity is a structure
```
                          "valueQuantity":
                            {
                                "value": 365
                            }
```

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/cb5654cbad59f847faf8336eb0c3b094e8ca7d78